_layouts/my-uw-template.html

```diff
@@ -2,7 +2,7 @@
 layout: default
 ---
 
-<div class="container-fluid my-uw">
+<div class="container-fluid my-uw" style="margin-top:60px;">
```

Comment:
Do you want to remove this inline style and just add `.navbar-fixed-top + .container-fluid` [here](https://github.com/UWMadisonUcomm/uw-ui-toolkit/blob/master/src/less/uw-ui-toolkit/uw-navbar.less#L45)?

It would be good for the Toolkit to handle `.container` and `.conainer-fluid` consistently.